src/SoftwareVersion.php

```diff
@@ -329,61 +323,55 @@ public static function showForSoftware(Software $soft)
-            echo "<th>" . __('Comments') . "</th>";
-            echo "</tr>\n";
-
-            $tot = 0;
-            foreach ($iterator as $data) {
-                Session::addToNavigateListItems('SoftwareVersion', $data['id']);
-                $nb = Item_SoftwareVersion::countForVersion($data['id']);
-
-                echo "<tr class='tab_bg_2'>";
-                echo "<td><a href='" . SoftwareVersion::getFormURLWithID($data['id']) . "'>";
-                echo $data['name'] . (empty($data['name']) ? "(" . $data['id'] . ")" : "") . "</a></td>";
-                echo "<td>" . $data['sname'] . "</td>";
-                echo "<td>" . Dropdown::getDropdownName(
-                    'glpi_operatingsystems',
-                    $data['operatingsystems_id']
-                );
-                echo "</td>";
-                echo "<td>{$data['arch']}</td>";
-                echo "<td>$nb</td>";
-                echo "<td>" . nl2br($data['comment'] ?? "") . "</td></tr>\n";
-
-                $tot += $nb;
-            }
-
-            echo "<tfoot>";
-            echo "<tr class='tab_bg_1 noHover'><td class='right b' colspan='4'>" . __('Total') . "</td>";
-            echo "<td class='b'>$tot</td><td></td>";
-            echo "</tr>";
-            echo "</tfoot>";
-            echo "</table>";
-        } else {
-            echo "<table class='tab_cadre_fixe'>";
-            echo "<tr><th>" . __('No item found') . "</th></tr>";
-            echo "</table>\n";
+        $tot = 0;
+        $entries = [];
+        $sv = new self();
+        foreach ($iterator as $data) {
+            $sv->getFromResultSet($data);
+            $nb = Item_SoftwareVersion::countForVersion($data['id']);
+
+            $tot += $nb;
+            $entries[] = [
+                'itemtype' => self::class,
+                'id' => $sv->getID(),
+                'version' => $sv->getLink(),
+                'status' => $data['sname'],
+                'os' => Dropdown::getDropdownName('glpi_operatingsystems', $data['operatingsystems_id']),
+                'arch' => $data['arch'],
+                'installations' => $nb,
+                'comments' => nl2br(htmlspecialchars($data['comment'] ?? ''))
+            ];
         }
 
-        echo "</div>";
+        TemplateRenderer::getInstance()->display('components/datatable.html.twig', [
+            'is_tab' => true,
+            'nopager' => true,
+            'nofilter' => true,
+            'nosort' => true,
+            'columns' => [
+                'version' => self::getTypeName(Session::getPluralNumber()),
+                'status' => __('Status'),
+                'os' => OperatingSystem::getTypeName(1),
+                'arch' => _n('Architecture', 'Architectures', 1),
+                'installations' => _n('Installation', 'Installations', Session::getPluralNumber()),
+                'comments' => __('Comments')
+            ],
+            'formatters' => [
+                'version' => 'raw_html',
```

Comment:
Result of `'comments' => nl2br(htmlspecialchars($data['comment'] ?? ''))` must not be escaped.

```suggestion
                'version' => 'raw_html',
                'comments' => 'raw_html',
```